app-react/src/components/Header.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import stickyNote from '../images/sticky_note-white.svg';
+
+const HeaderStyled = styled.header`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 2;
+  position: relative;
+  background: var(--main-b-color);
+  padding: 15px 20px;
+  box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.07), 0 3px 10px 0 rgba(0, 0, 0, 0.07);
+  color: white;
+  min-height: 50px;
+  max-height: 50px;
+`;
+
+const HeaderTitle = styled.div`
+  display: flex;
+  h1 {
+    font-size: 28px;
+    margin: 0;
+  }
+  img {
+    width: 24px;
+    margin-left: 6px;
+    margin-bottom: -6px;
+  }
+`;
+
+const ProfileContainer = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const SignoutButton = styled.button`
+  background: white;
+  color: var(--main-b-color);
+  border-radius: 3px;
+  font-size: 14px;
+  font-weight: 700;
+`;
+
+const AvatarImg = styled.img`
+  height: 50px;
+  width: 50px;
+  border-radius: 53px;
+  margin-left: 20px;
+  box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.07), 0 3px 10px 0 rgba(0, 0, 0, 0.07);
+`;
+
+const Header = ({ profileVisible, avatarImgPath }) => {
+  return (
+    <HeaderStyled class='header'>

```

Comment:
In React the `class` attribute needs to be `className`. It still works but it will throw an Invalid DOM property warning in the console.